cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -2862,6 +2863,49 @@ deserialiseFromBech32 asType bech32Str = do
 
     return value
 
+deserialiseAnyOfFromBech32
+  :: [FromSomeType SerialiseAsBech32 b]
+  -> Text
+  -> Either Bech32DecodeError b
+deserialiseAnyOfFromBech32 types bech32Str = go [] types
+  where
+    go
+      :: [Bech32DecodeError]
+      -> [FromSomeType SerialiseAsBech32 b]
+      -> Either Bech32DecodeError b
+    go errs (FromSomeType ttoken fromType:ts) =
```

Comment:
Seems like `errs` accumulates data, but ultimately always gets thrown away?

Reply:
It isn't always thrown away, see the `go errs []` case.

Reply:
I've made some changes to this function, so marking this thread as "resolved".